### PR TITLE
Improve table loading popup 

### PR DESCRIPTION
**Pull request by TruongQuangSB**
- Avoid each Table create own pop up, when table view update

### CODE REVIEW

File: java/bundles/org.eclipse.set.model.titlebox/src/org/eclipse/set/model/titlebox/PlanningOffice.java
Comment (mariusheine) [inline java/bundles/org.eclipse.set.model.titlebox/src/org/eclipse/set/model/titlebox/PlanningOffice.java:1]: @TruongQuangSB The changes in this file seems to be unrelated to me. Where does it come from?

File: java/bundles/org.eclipse.set.model.titlebox/src/org/eclipse/set/model/titlebox/StringField.java
Comment (mariusheine) [inline java/bundles/org.eclipse.set.model.titlebox/src/org/eclipse/set/model/titlebox/StringField.java:1]: @TruongQuangSB Here as well

**Review by mariusheine — COMMENTED** [review]
@TruongQuangSB There are a lot of other files where unrelated changes are inside. I did not have a look in all these files. Can we maybe move them in a separate PR with the context `clean up` or something like this.

The changes affecting the table service and table views (so the actual feature) looks pretty nice to me.

**Comment by mariusheine** [comment]
@TruongQuangSB And can you please fix the typo in the PR title ;)

**Comment by TruongQuangSB** [comment]
> @TruongQuangSB There are a lot of other files where unrelated changes are inside. I did not have a look in all these files. Can we maybe move them in a separate PR with the context `clean up` or something like this.
> 
> The changes affecting the table service and table views (so the actual feature) looks pretty nice to me.

@mariusheine ok, i have move the checkstyle fix to another PR

**Comment by mariusheine** [comment]
> > @TruongQuangSB There are a lot of other files where unrelated changes are inside. I did not have a look in all these files. Can we maybe move them in a separate PR with the context `clean up` or something like this.
> > The changes affecting the table service and table views (so the actual feature) looks pretty nice to me.
> 
> @mariusheine ok, i have move the checkstyle fix to another PR

Perfect :+1:
